### DISCUSSION TITLE
Hades Canyon NUC: Retrospective

### DISCUSSION THREAD

**Discussion by osy** [osy86/HaC-Mini#578] [General]
https://osy.gitbook.io/hac-mini-guide/details/hades-canyon-nuc

I wrote a new (and probably final) article summarizing the history of the Hades Canyon NUC and HaC mini. With this and the v3.0 release of HaC mini, the project is mature enough to last until Apple stops releasing x86 builds of macOS. I will continue to do driver updates (planned 1-2 times a year) and bug fixes up until I stop using this NUC (which I don't plan on doing anytime soon). Once I do leave, I will hand the repository over to people who have made contributions, bug fixes, and pull requests in the past. I am quite disappointed that Intel did not release a successor to Hades Canyon that is [competitive](https://github.com/osy/HaC-Mini/discussions/557) but I look forward to moving to Apple Silicon in the future.

**Comment by ninhxuanluu** [comment]
As a Hades Canyon user, thank you so much <3

**Comment by Malumen** [comment]
Hi Osy,

Thank you so much for all your hard work. I really do wish I knew as much as you about the software development. I always thought what a super cool thing it is to be able to make a dynamic installer package!

I look forward to your updates soon! I had hoped to see the `HaC mini Project` last until the OpenCore 1.0.0 release, oh well.

On the note of Intel's latest NUC series: _agreed_. Pretty disappointing. I was hoping for a 9750H or better custom silicon. There was no way Intel would pair up with AMD Radeon again, so at least the _idea_ of the Intel Compute Unit was cool (PCIe style card that can be slotted into another PCIe riser board for additional devices like GPUs and SSDs)...

Too bad future similar products will be too far gone to spoof macOS compatible CPUs... Bummer.

I'm looking to save and buy an `MSi Z590i UNIFY` motherboard and a `10700k` in the hopes to have a longterm high end Work/Play Mac/Windows system. Also you may be intrigued to find out that there is huge progress with the new `Maple Ridge` Thunderbolt 4 controller found on these new motherboards!


In any case, best of luck and good health to you in your future endeavours Osy!!!

**Comment by controlfix87** [comment]
THANKS A LOT!!!! 
WOW what a great project! you are highly skillable :)

**Comment by xiaooloong** [comment]
🎉 谢谢你为「冥王峡谷」NUC 的付出，使它成为了我用过最好的 Hackintosh。👍

**Comment by icespring** [comment]
Thank you so much for the best NUC

**Comment by chaoyangnz** [comment]
This was why I bought NUC Hades years ago.
It is working pretty fine.

**Comment by yuchong-li** [comment]
Thank you so much OSY!!!

**Comment by JasonLamv-t** [comment]
Thank you so much!

**Reply by controlfix87** [reply]
Is there a way to downgrade to Big Sur?

Thanks :)

**Reply by osy** [reply]
Please make a new thread.

**Comment by alenzhong** [comment]
Thank you so much!!!
Brings a lot of convenience to my work!

**Comment by Juzi1ng** [comment]
thanks

**Comment by kevinpvo** [comment]
I just want to say thank you so much to you @osy!

This project has been by far the best thing that has happened for this NUC and it really wouldn't be the same without you! The amount of work and effort you have put into the hac-mini has made this the most complete and thorough hackintosh system I have ever come across, and it was the main driving factor in my decision to purchase the hades canyon!

**Comment by osy** [comment]
https://www.servethehome.com/intel-exiting-the-pc-business-as-it-stops-investment-in-the-intel-nuc/

**Comment by woshilaiba** [comment]
Thank you ! sincerely!

**Comment by pgou** [comment]
Thank you so much!
